addons/transform/org.openhab.transform.feed.test/FeedTransformationServiceTest.launch

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Yes, we usually include them, because it is the easiest way to run the tests from within the IDE - constructing a launch config yourself can be very cumbersome.